devices/common_patches/l4_adc_common.yaml

```diff
@@ -0,0 +1,166 @@
+_add:
+  # This SVD is missing the ADC_Common peripheral that most other parts with
+  # this ADC contain; consequently it's missing the CSR and CCR and CDR regs
+  # from RM0394.
+  ADC_Common:
+    description: ADC common registers
+    groupName: ADC
+    baseAddress: 0x50040300
+    addressBlock:
+      offset: 0
+      size: 0xC
+      usage: registers
+    registers:
+      CSR:
+        description: ADC common status register
+        addressOffset: 0x0
+        access: read-only
+        resetValue: 0x00000000
+        fields:
+          JQOVF_SLV:
```

Comment:
As far as I can tell, none of the L4x1/2/3 chips (i.e. any covered by RM0394) actually contain any of the `_SLV` fields, it looks like they're only present on the L4x5?

Reply:
From what I find, only the L41x should be like this? See `Table 54`.

Reply:
Oh, I see what's happened; I have an older copy of RM0394 from when it covered L43/L43/L45/L46 and at the time L4x1 was covered by a separate RM0392; the latest revision of RM0394 covers L41x/L42x/L43x/L44x/L45x and RM0351 covers the L47x/L48x/L49x/L4Ax. Annoyingly it means the division of SVD files doesn't actually match the division of datasheets, go figure.

Anyway now that I'm using the latest RM0394 everything makes sense and I think we can go ahead with all the _SLV bits too.